### PR TITLE
Setup webpack and data

### PR DESCRIPTION
**Pull request by akhror-valiev**
- The web page will be built by webpack and used as a template using the HTML Webpack Plugin.

### CODE REVIEW

**Comment by akhror-valiev** [comment]
# Summary
 - Web page will be built using webpack
 - Used a template using the HTML Webpack Plugin

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @akhror-valiev 
I'm sure you put a lot of effort into this project but this PR is invalid because you didn't setup a README file and you don't have a pull request description😓

- [ ] Kindly write the README file using this [template](https://github.com/microverseinc/readme-template)

- [ ] Kindly copy the text you wrote in the comment to the pull request description (click on the 3 dots then `edit`) 👍 

Kindly fix the issues and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                <!-- <li class="input-li">
+                    <label class="input-label" for="">
+							<input type="checkbox" />
+							<p>DO some</p>
+
+							<span class="menu-icon"
+								><i class="fa-solid fa-ellipsis-vertical"></i
+							></span>
+						</label>
+                </li> -->
```
Comment (Abenezer-Tilahun) [inline src/index.html:32]: - [ Optional ] Kindly remove the commented codes from your code-base ,the commented code adds confusion with no real benefit and makes your code-base unclean .

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  {
+    description: 'online meeting',
+    completed: true,
+    index: 1,
+  },
+];
+const todoWrapper = document.querySelector('.new-task');
+// const newToDo = document.querySelector('.input-text');
+// const addToDo = document.querySelector('.btn-add');
+// const info = [];
```
Comment (Abenezer-Tilahun) [inline src/index.js:35]: - [ Optional ] Kindly remove the commented codes from your code-base ,the commented code adds confusion with no real benefit and makes your code-base unclean .

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @akhror-valiev    
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**